crates/storage_models/src/reverse_lookup.rs

```diff
@@ -13,10 +13,10 @@ use crate::schema::reverse_lookup;
 pub struct ReverseLookup {
     /// Primary key. The key id.
     pub lookup_id: String,
-    /// the value id. i.e the id you want to access KV table.
-    pub pk_id: String,
     /// the `field` in KV database. Which is used to differentiate between two same keys
     pub sk_id: String,
+    /// the value id. i.e the id you want to access KV table.
+    pub pk_id: String,
```

Comment:
Whoa, good catch. Thanks!

We'll need to seriously find reliable ways of catching such mistakes/differences with field ordering in struct definition against that in `schema.rs`.

Reply:
Hey, not sure if this is helpful or not, but I was able to find

https://docs.rs/diesel-sort-struct-fields/latest/diesel_sort_struct_fields/

Maybe we can look into it....